src/main/java/ua/mykolamurza/chatullo/Command/MuteCommand.java

```diff
@@ -0,0 +1,44 @@
+package ua.mykolamurza.chatullo.Command;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public class MuteCommand implements CommandExecutor {
+
+    private final JavaPlugin plugin;
+
+    public MuteCommand(JavaPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (args.length == 0) {
+            sender.sendMessage("Usage: /mute <player>");
+            return true;
+        }
+
+        String targetPlayerName = args[0];
+        writeMutedPlayerToFile(targetPlayerName);
```

Comment:
Who has to be able to mute the player? Everyone, console, or only Operators? Looks like everyone can mute everybody.